src/main/java/reposense/parser/GroupConfigCsvParser.java

```diff
@@ -23,8 +23,13 @@
     private static final int GROUP_NAME_POSITION = 1;
     private static final int FILES_GLOB_POSITION = 2;
 
+    /**
+     * Number of columns expected in the header of repo-config.csv file.
+     */
+    private static final int HEADER_SIZE = 3;
```

Comment:
can we do something like `EXPECTED_HEADER_SIZE = FILES_GLOB_POSITION + 1 // last position + 1`?

I would prefer it to be together with all the `POSITION` constants so that no one will forget to update it.

Reply:
Done.

Reply:
Either is fine with me, but all the comments e.g.
```
    // Number of columns expected in the header of repo-config.csv file.
```
is rather redundant to me as this comment should be in the header comments of `CsvParser` instead.

Do add the inline comment `// last position + 1` so that subsequent developers would be reminded to update it

Reply:
Done. I've added the inline comment `// last position + 1` 
On second thought, perhaps we do not need a header comment in `CsvParser` as the attribute name (`expectedHeaderSize`) is self explanatory.